app/templates/components/tabbed-navigation.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if device.isMobile}}
-  <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
+  <div role="button" class="ui segment center aligned" onClick={{action 'toggleMenu'}}>
```

Comment:
Why is `onClick=` needed ? An action by default binds to the click event.

Reply:
@niranjan94 The value is not getting updated on simply using action because we're using the jquery object `event` in `tabbed-navigation.js` which we can only pass via `onClick` method. So if I will remove the `onClick` the jquery object will not be passed and we won't be able to set it. Also, I am using the jquery object because the action lies on the dropdown and not individual items so for that we need the event which has the info about what was clicked which I am fetching using `event.srcElement.text`.

Reply:
No. `event` is passed in even when using a normal action. `event` will directly be accessible within the action method. (Don't declare an argument)

Reply:
@niranjan94 ok! Passing argument was causing the issue. I have removed it and made changes. Now it's working as expected. Can you please review again? Thanks.